public_html/src/modules/Core/modules/Users/router/routes.js

```diff
@@ -11,19 +11,19 @@ export default [
       {
         name: 'Core.Users.Login.LoginForm',
         path: 'form',
-        meta: { module: 'Core.Users', view: 'Login' },
+        meta: { module: 'Users' },
```

Comment:
Should be Core.Users not Users because of Hierarchy - Users can be in Base or Settings category too